shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -0,0 +1,317 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_view_elementary.h"
+
+#include "flutter/shell/platform/tizen/flutter_tizen_view.h"
+#include "flutter/shell/platform/tizen/logger.h"
+
+#include <efl_extension.h>
+#include <ui/efl_util.h>
+
+namespace {
+
+static const int kScrollDirectionVertical = 0;
+static const int kScrollDirectionHorizontal = 1;
+static const int kScrollOffsetMultiplier = 20;
+
+}  // namespace
+
+namespace flutter {
+
+TizenViewElementary::TizenViewElementary(TizenBaseHandle::Geometry geometry,
+                                         Evas_Object* elm_parent)
+    : TizenView(geometry), elm_parent_(elm_parent) {
+  if (!CreateWindow()) {
+    FT_LOG(Error) << "Failed to create a platform window.";
+    return;
+  }
+
+  SetWindowOptions();
+  RegisterEventHandlers();
+  Show();
+}
+
+TizenViewElementary::~TizenViewElementary() {
+  UnregisterEventHandlers();
+  DestroyWindow();
+}
+
+bool TizenViewElementary::CreateWindow() {
+  elm_config_accel_preference_set("hw:opengl");
+
+  Ecore_Evas* ecore_evas =
+      ecore_evas_ecore_evas_get(evas_object_evas_get(elm_parent_));
+
+  elm_win_ = elm_win_get(elm_parent_);
```

Comment:
`elm_win_get` is not public API. maybe you need to find some other way...

Reply:
You are right. I will fix all related to `elm_win_`.

Reply:
I first removed the unnecessary window code. please check again